src/tests/ftest/nvme/nvme_io_stats.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+import re
+from ior_test_base import IorTestBase
+from general_utils import get_log_file, run_task
+
+#List of NVMe statistics
+NVME_STATS = ['read_bytes',
+              'read_ops',
+              'write_bytes',
+              'write_ops',
+              'read_latency_ticks',
+              'write_latency_ticks']
+
+class NvmeIOStates(IorTestBase):
+    # pylint: disable=too-many-ancestors
+    """Test class Description: Runs IOR with 1 server with basic parameters and
+       verify NVMe IO statistics parameters getting increase.
+
+    :avocado: recursive
+    """
+
+    def test_nvme_io_stats(self):
+        """Jira ID: DAOS-4722.
+
+        Test Description:
+            Purpose of this test is to run IO test and check when NVME_IO_STATS
+            enabled in config, it generates the different statistics.
+
+        Use case:
+            Run ior and it will print the NVMe IO stats to control plane log
+            file.
+
+        :avocado: tags=all,hw,medium,nvme,ib2,nvme_io_stats,full_regression
+        """
+        transfer_block_size = self.params.get("transfer_block_size",
+                                              '/run/ior/iorflags/*')
+
+        # Update IOR parameter
+        self.ior_cmd.flags.update(self.params.get("ior_flags",
+                                                  '/run/ior/iorflags/*'))
+        self.ior_cmd.daos_oclass.update(self.params.get("obj_class",
+                                                        '/run/ior/iorflags/*'))
+        self.ior_cmd.api.update(self.params.get("ior_api",
+                                                '/run/ior/iorflags/*'))
+        self.ior_cmd.transfer_size.update(transfer_block_size[0])
+        self.ior_cmd.block_size.update(transfer_block_size[1])
```

Comment:
Apologies for not pointing it out in my previous review.
Optional: These lines can avoided by supplying the corresponding in-built param names for these arguments.

Reply:
I already took care of it in my current patch